ci/docker-publish.yml

```diff
@@ -90,4 +90,4 @@ jobs:
         # This step uses the identity token to provision an ephemeral certificate
         # against the sigstore community Fulcio instance, and records it to the
         # sigstore community Rekor transparency log.
-        run: cosign sign ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}@${{ steps.build-and-push.outputs.digest }}
+        run: cosign sign ${{ steps.meta.outputs.tags }}@${{ steps.build-and-push.outputs.digest }}
```

Comment:
Could it be a problem if `tags` points to a list of tags?

Reply:
Given line 66, I don't think that's possible today, but it could be a problem as folks alter the workflow.